docs/src/main/asciidoc/rest-client.adoc

```diff
@@ -141,6 +141,24 @@ The default scope can also be defined on the interface.
 
 Note that `org.acme.restclient.CountriesService` _must_ match the fully qualified name of the `CountriesService` interface we created in the previous section.
 
+To facilitate the configuration, you can use the `@RegisterRestClient` `configKey` property that allows to use another configuration root than the name of your interface.
```

Comment:
This should be more descriptive than just saying "name", as it's actually replacing the full package and class name

Reply:
Is `the full qualified name of your interface` better ?
The example is self explainatory :)

Reply:
That works

Reply:
done